### ISSUE TITLE
list of annotation capabilities?

### ISSUE THREAD

**Issue by mrtristan** [domaindrivendev/Swashbuckle#495]
was wondering if there is a full list of capabilities as to what Swashbuckle can handle when it comes to annotations and getting things in to the front end. after searching around for other things i've noticed comments like this:

https://github.com/domaindrivendev/Swashbuckle/issues/237#issuecomment-100789024

that i otherwise would not have known about. is there any chance of getting this section:
https://github.com/domaindrivendev/Swashbuckle#including-xml-comments
updated so we can fully utilize the capabilities of what you've built here? i'm getting in to really customizing the front end so i'm starting to dig deeper and don't want to reinvent the wheel since you've already spent a lot of time in here already and probably have a few more tricks that i dont know about... :+1:

**Comment by Corendiel** [comment]
I agree the list of  the followings don't looks like a list of Xml Tags

Action summary
Action remarks
Parameter summary
Type summary
Property summary
